test/unit/release-tests.sh

```diff
@@ -105,6 +107,11 @@ test_function ${SUCCESS} ":knative-nightly/test-infra:" call_function_post "echo
 test_function ${SUCCESS} ":foo:" call_function_post "echo :\$KO_DOCKER_REPO:" parse_flags --release-gcr foo --publish
 test_function ${SUCCESS} ":foo:" call_function_post "echo :\$RELEASE_GCS_BUCKET:" parse_flags --release-gcs foo --publish
 
+echo ">> Testing ACR/BLOB values"
+
+test_function ${SUCCESS} "Not publishing the release, GCR/ACR flags ignored" parse_flags --release-acr foo
+test_function ${SUCCESS} "Not publishing the release, GCS/BLOB flags ignored" parse_flags --release-azblob foo
+
```

Comment:
Note that no test was added to actually go through your new code, so it's pretty easy to break it in future changes and never notice until someone actually tries to deploy to Azure. I strongly recommend you add such tests.

Reply:
Any suggestions on what kind of tests could be added... cant figure how to add more unit coverage...?

Reply:
At a minimum, test the Azure-exclusive functions you added.

Reply:
Have tested them manually - not possible to add a test to this repo though since that would require hitting azure resources

Reply:
We don't run E2E tests here. Thus, mock the tools you're using and write relevant tests, just like it's being done in this very file for hub or git. Once again, a single manual test won't protect these features from future unintended breakage since they are not used regularly.

Reply:
added tests!